src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,27 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
```

Comment:
```suggestion
* Broker creation is managed by the operator
```
You're saying opposite things at the same time :) I understand the user interface remains in subctl :) but sounds clearer without that part.

Reply:
I guess she wanted to say "instead"

Reply:
> I guess she wanted to say "instead"

yes. And edited accordingly